kleva-j/iReporter
# Feature/#162736990/delete a record from frontend

#### What does this PR do?
- Ensures an authenticated user records are deleted from the frontend

#### Description of the task to be completed
 - create functions to make requests to the API from the frontend to delete a record.

#### How should this be manually tested?
- Clone the repository
- Check out into the branch Feature/#162736990/delete a record from the frontend
- Install all dependencies
- run `npm start` to run the app successfully.

#### What are the relevant pivotal tracker stories?
[162736990](https://www.pivotaltracker.com/story/show/162736990)
